.github/workflows/windows_builds.yml

```diff
@@ -70,15 +80,20 @@ jobs:
         with:
           repo: godotengine/godot-angle-static
           version: tags/chromium/6029
-          file: Windows.6029-1.MSVC_17.x86_64.x86_32.zip
+          file: Windows.6029-1.${{ matrix.compiler == 'gcc' && 'MinGW_11' || 'MSVC_17' }}.x86_64.x86_32.zip
```

Comment:
This build is a bit outdated actually, would be good to update to https://github.com/godotengine/godot-angle-static/releases/tag/chromium%2F6601.2

Could be done in a separate PR though as it's not directly related to the change here. But it will impact this PR too as the file naming changed, so we might as well do it together.